Dockerfile

```diff
@@ -1,212 +1,38 @@
-        --backup=no \
-        --deldoc=yes \
-        --fstrans=no \
-        --default
-
-ENV XVIDCORE_VERSION 1.3.2
-RUN curl -sSL http://mirror.ryansanden.com/ffmpeg-d00bc6a8/xvidcore-${XVIDCORE_VERSION}.tar.gz | \
-        tar -xz \
-    && cd xvidcore/build/generic \
-    && ./configure --prefix='/usr/local' \
-    && make \
-    && make install
-
-#ENV FFMPEG_COMMIT 580c86925ddf8c85d2e6f57ed55dd75853748b29
-RUN git clone git://source.ffmpeg.org/ffmpeg.git \
-    && cd ffmpeg \
-    && git checkout release/2.4 \
-#&& git checkout -q $FFMPEG_COMMIT \
-    && ./configure \
-        --enable-shared \
-        --enable-gpl \
-        --enable-libfaac \
-        --enable-libmp3lame \
-        --enable-libopencore-amrnb \
-        --enable-libopencore-amrwb \
-        --enable-librtmp \
-        --enable-libtheora \
-        --enable-libvorbis \
-        --enable-libx264 \
-        --enable-nonfree \
-        --enable-version3 \
-        --enable-libxvid \
-    && make \
-    && make install \
-    && ldconfig
-##########
-
-WORKDIR /opt/ori
-# Create a virtualenv project
-RUN echo 'ok'
-RUN virtualenv -q /opt
-RUN echo "source /opt/bin/activate; cd /opt/ori;" >> ~/.bashrc
-
-# Temporarily add all NPO Backstage files on the host to the container
-# as it contains files needed to finish the base installation
-ADD . /opt/ori
-
-# Install Python requirements
-RUN source ../bin/activate \
-    && python -m pip install -U pip \
-    && pip install Cython==0.21.2 \
-    && pip install -r requirements.txt
-
-# Start
-RUN source ../bin/activate \
-    && service elasticsearch restart \
-    && sleep 20 \
-    && ./manage.py elasticsearch create_indexes es_mappings/ \
-    && ./manage.py elasticsearch put_template
-
-RUN apt-get install supervisor
-
-# Delete all NPO Bastage files again
-RUN find . -delete
-
-# When the container is created or started run start.sh which starts
-# all required services and supervisor which starts celery and celerycam
-# CMD /opt/ori/start.sh
+FROM python:2.7-alpine
+MAINTAINER Jurrian Tromp <jurrian@argu.co>
+
+# Install system requirements
+RUN apk add --update build-base git tzdata libxml2-dev libxslt-dev ffmpeg-dev \
+  fontconfig-dev jpeg-dev openjpeg-dev zlib-dev openssl-dev libffi-dev inotify-tools libmagic \
+  && pip install cython
+
+# Install python requirements
+COPY ocd_backend/requirements.txt /opt/ori/requirements.txt
+RUN pip install --no-cache-dir -r /opt/ori/requirements.txt
+
+# Install poppler and pdfparser
+COPY src/poppler /tmp/poppler
+WORKDIR /tmp/poppler/
+RUN ./configure --disable-poppler-qt4 --disable-poppler-qt5 --disable-poppler-cpp --disable-gtk-test --disable-splash-output --disable-utils \
+  && make \
+  && make install
+
+RUN git clone https://github.com/izderadicka/pdfparser.git /tmp/pdfparser
+WORKDIR /tmp/pdfparser/
+RUN ldconfig /tmp/pdfparser \
+  && POPPLER_ROOT=/tmp/poppler python setup.py install
+
+# Cleanup
+RUN pip uninstall -y cython \
```

Comment:
Cython is needed in the installation of some packages, but isn't used in the code. Uninstalling makes a smaller docker image :)